src/braket/default_simulator/density_matrix_simulation.py

```diff
@@ -288,8 +285,8 @@ def _apply_kraus(
             _, needs_swap = multiply_matrix(
                 result, superop, targets_new, out=temp, return_swap_info=True, dispatcher=dispatcher
             )
-            if needs_swap:
-                result, temp = temp, result
+            # With gate_type dispatch, swaps won't occur. An optimization would be to do is add matrix matching to avoid genearl 1q, 2q cases.
```

Comment:
```suggestion
            # With gate_type dispatch, swaps won't occur. An optimization would be to do is add matrix matching to avoid general 1q, 2q cases.
```